src/main/java/to/us/resume_builder/resume_components/category/HeaderCategory.java

```diff
@@ -31,34 +31,66 @@ public HeaderCategory(String id) {
         super(id, CategoryType.HEADER);
     }
 
+    /**
+     * Returns the current String link
+     * @return link
+     */
     public String getLink() {
         return link;
     }
 
+    /**
+     * Sets the value of link
+     * @param link
+     */
     public void setLink(String link) {
         this.link = link;
     }
 
+    /**
+     * Returns the current String email
+     * @return email
+     */
     public String getEmail() {
         return email;
     }
 
+    /**
+     * Sets the value of email
+     * @param email
+     */
     public void setEmail(String email) {
         this.email = email;
     }
 
-    public String getPhone_number() {
+    /**
+     * Returns the current String phone number
+     * @return the current String phone_number
+     */
+    public String getPhoneNumber() {
         return phone_number;
```

Comment:
Change the `phone_number` field to be camelCase (i.e. `phoneNumber`).

Reply:
(applies to whole class)

Reply:
Done